### PR TITLE
[13.0][FIX] web_company_color: Recalculated color when we do not want

### PR DESCRIPTION
**Pull request by CarlosRoca13**
If we change something with base.document.layout wizard, this will launch the colors computing even without changing the logo. This procedure will change the value of the colors if they were set by hand.

This change forces the user to press a button to recompute the colors and avoid possible undesired changes.

cc @Tecnativa TT34902

please @pedrobaeza @Tardo review this

### CODE REVIEW

**Review by Tardo — COMMENTED** [review]
Looks good.. but for some reason its not working on runboat... Can you confirm that it is something related to runboat?

File: web_company_color/models/res_company.py [hunk 1]
```diff
@@ -74,34 +74,35 @@ def write(self, values):
...
             result = super().write(values)
-
             if any([field in values for field in fields_to_check]):
                 self.scss_create_or_update_attachment()
         else:
             result = super().write(values)
         return result
 
+    def button_compute_color(self):
+        values = self.default_get(
```
Comment (pedrobaeza) [inline web_company_color/models/res_company.py:86]:
```suggestion
        self.ensure_one()
        values = self.default_get(
```

File: web_company_color/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -2,3 +2,7 @@
 * Lois Rilo <lois.rilo@forgefloww.com> (ForgeFlow)
 * Simone Orsi <simone.orsi@camptocamp.com>
 * Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (pedrobaeza) [inline web_company_color/readme/CONTRIBUTORS.rst]: Group this one as well

**Comment by pedrobaeza** [comment]
> Looks good.. but for some reason its not working on runboat... Can you confirm that it is something related to runboat?

Can you be more specific? What is not working?

**Comment by Tardo** [comment]
> > Looks good.. but for some reason its not working on runboat... Can you confirm that it is something related to runboat?
> 
> Can you be more specific? What is not working?

The color fields update but when I refresh the page I don't see those colors applied to the navbar. In runboat 14.0 I see it updates, but in this PR and 13.0 runboat seems not to update the css.

File: web_company_color/models/res_company.py [hunk 3]
```diff
@@ -74,34 +74,35 @@ def write(self, values):
...
-                            "color_navbar_bg_hover": n_rgb_to_hex(_rd, _gd, _bd),
-                            "color_navbar_text": "#000" if _a < 0.5 else "#fff",
-                        }
-                    )
-                else:
-                    values.update(self.default_get(fields_to_check))
-
             result = super().write(values)
-
             if any([field in values for field in fields_to_check]):
```
Comment (pedrobaeza) [inline web_company_color/models/res_company.py:78]: These 2 lines should be moved to the new method.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Problem seen

File: web_company_color/models/res_company.py (hunk 3 above)
Comment (CarlosRoca13) [reply web_company_color/models/res_company.py:78]: I don't think that's the problem, locally is working well, btw the new method is launching a write with the values computed

Could be that the context on runboat has the key ignore_company_color?
https://github.com/OCA/web/pull/2157/files#diff-9b4dca5f4bdbc5450fc14eeba219185f4894189e94d026978af5a65841c5f85eR71

Comment (pedrobaeza) [reply web_company_color/models/res_company.py:78]: Yes, you are right that it's called anyway, as after the write method is there.

Comment (pedrobaeza) [reply web_company_color/models/res_company.py:78]: I don't see the problem right now.

Comment (CarlosRoca13) [reply web_company_color/models/res_company.py:78]: I test it on runboat and it's working

@Tardo, did you test it with the correct company?

**Comment by CarlosRoca13** [comment]
Other changes done :+1:

**Comment by pedrobaeza** [comment]
I don't understand what is happening:

Selección_041

**Comment by CarlosRoca13** [comment]
> I don't understand what is happening:
> 
> Selección_041

That was a test done by me :sweat_smile: 

Try it again please

**Comment by CarlosRoca13** [comment]
BTW you are changing it on "My Company (Chicago)" but you are at "My Company (San Francisco)"

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Oh, true. Merging as working:

/ocabot merge minor

Please fw-port it to v14

**Pull request merged**